scripts/setup-local-cluster.sh

```diff
@@ -175,13 +176,13 @@ function setup_node() {
       --init \
       --password="${password}" \
       --api \
-      --apiHost "${host}" \
+      --apiHost "0.0.0.0" \
```

Comment:
I'm not a fan of using all IPs. What's the reason for this change?

Reply:
The API of the nodes running in pluto had the value: `127.0.0.1` which makes the API not accessible from outside the docker container. I had to open it to `0.0.0.0` to make them visible from the host or from other containers (running in docker-compose).

I needed because the docker-compose used in load testing, needs to access the API of the nodes. 
https://github.com/hoprnet/hoprd-test/blob/main/k6/docker-compose.yaml

Reply:
Ok, but then we should just fix the pluto arguments to use the correct host instead and not hard-code it there.

Reply:
Of course, I didn't realised that there was an input parameter to modify that value. I've reverted it